CLAUDE.md

```
@@ -95,12 +95,32 @@ The application consists of five main services that communicate via queues:
 ### Supporting Infrastructure
 - **MySQL** (port 3306): Primary data store for local dev
 - **Minio** (port 9999): S3-compatible blob storage for local dev
-- **NATS** (port 4222): Message queue for local dev
+- **NATS** (port 4222): Message queue for distributed mode
+- **Channels**: In-memory Go channels for all-in-one mode
 
 ### Production Alternatives
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)